quantuminspire/util/configuration.py

```diff
@@ -20,15 +20,7 @@
 
 DEFAULT_CONFIG = """
 {
-  "auths": {
-    "https://staging.qi2.quantum-inspire.com": {
-      "client_id": "Yz7ni9PUAyT43eUASZfmc1yqI66QxLUJ",
-      "well_known_endpoint":  "https://quantum-inspire-staging.eu.auth0.com/.well-known/openid-configuration"
-    },
-    "https://api.qi2.quantum-inspire.com": {
-      "well_known_endpoint":  "https://auth.qi2.quantum-inspire.com/realms/oidc_production/.well-known/openid-configuration"
-    }
-  }
+  "auths": {}
```

Comment:
Should the `DEFAULT_CONFIG` even be here?

Reply:
It is being used as skeleton for the auth_config file if this file had not yet been created (see [ensure_config_file_exists](https://github.com/QuTech-Delft/quantuminspire2/blob/4523f758ad5814f81b21c0e2ac3c62b5eced5f30/quantuminspire/util/configuration.py#L28) method). Later on when we want to populate it we expect at least the "auths" key to be present that's why we still need it.

Reply:
Ah nevermind I think we can drop it